README.md

```diff
@@ -66,13 +66,19 @@ The following steps are required to deploy the infrastructure from the command l
    az account set --subscription xxxxx
    ```
 
-1. Create a resource group and deploy the infrastructure.
-
+1. Set the deployment location. Because this solution uses Azure AI Studio, the location MUST be one of: `southcentralus`, `westeurope`, `southeastasia`, `japaneast` to support all resources deployed
```

Comment:
```suggestion
1. Set the deployment location.

   Because this solution uses Azure AI Studio, the location MUST be one of: `southcentralus`, `westeurope`, `southeastasia`, `japaneast` to support all resources deployed.

```